docs/user_guide/SIMP_Administration/Environments/Environments_Examples.inc

```diff
@@ -23,6 +23,62 @@ You might also find the following HOWTO documents to be helpful:
   is set up, the environment a node is assigned to can be set in using a configuration
   file located on the puppetserver.
 
+.. _ug-environments-deploying-from-local-repositories:
+
+Updating an Existing Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you are using the :ref:`Local deployment
+scenario<ug-sa-env-deployment-scenarios--local>`
+the following steps can be used to update and deploy puppet modules from
+the local git repositories to an existing environment or to update any
+of your site specific modules in your environment.
+
+#. **Update Site Specific Puppet Modules**
+
+   Follow your site procedures to update any of your site specific
+   puppet modules.
+
+   Make sure all your modules have an entry in the :file:`Puppetfile`.
+   See  :ref:`Edit the Puppetfile to include non-simp site modules<howto-generate-a-simp-puppetfile>`
+   for more details.
+
+#. **Update SIMP Local Git Repositories**
+
+   On the yum server (usually the SIMP master on smaller deployments)
+   add any new or updated SIMP module RPMs to the yum repositories using
+   your sites procedures.
+
+   On your SIMP master, clear the yum cache
+
+   .. code-block:: sh
+
+      yum clean all; yum makecache
+
+   Then install or update any modules
+
+   .. code-block:: sh
+
+      # For example if installing all the modules in simp-extras
+      yum install simp-extras
+      # or a simp puppet module with a patch
+      yum update pupmod-simp-aide
+
+   This will update the local git repositories on the puppetserver.
```

Comment:
Should this be SIMP master or SIMP server (should be consistent)?  Line 48 above talks about SIMP master.

Reply:
fixed